Misc/NEWS.d/next/Core and Builtins/2020-11-20-00-57-47.bpo-42195.HeqcpS.rst

```diff
@@ -0,0 +1,7 @@
+The ``__args__`` of the parameterized generics for :data:`typing.Callable`
+and :class:`collections.abc.Callable` are now consistent.  Said ``__args__`` 
+are now in the form ``(_PosArgs[args], result)``.  For example, 
+``typing.Callable[[int], float].__args__`` now gives 
+``(typing._PosArgs[int], float)``.  Due to this change, 
```

Comment:
This ignores the fact that typing._PosArg and collections.abc._PosArg are not the same object. (Not sure it's important, but the way it currently reads it can be misread to imply that.)

Also:
```suggestion
``(typing._PosArgs[int], float)``.  To allow to this change, 
```